packages/core/src/index.ts

```diff
@@ -850,19 +894,33 @@ class Hopr extends EventEmitter {
 
   // This is a utility method to wait until the node is funded.
```

Comment:
I don't think this comment no longer fits the purpose since the method will throw after MAX_DELAY

Reply:
updated